frontend/src/metabase/actions/containers/ActionCreator/ActionCreator.tsx

```diff
@@ -102,6 +113,7 @@ function ActionCreator({
 
   const query = question.query() as NativeQuery;
   const isNew = !action && !question.isSaved();
+  const isEditable = isNew || Boolean(model?.can_write);
```

Comment:
nit: maybe explicit form `|| (model != null && model.can_write)` would be easier to read?

Reply:
I don't have a strong preference, but I find it readable enough as is for now